### PR TITLE
Add Button to Jump to Table of Contents

### PR DESCRIPTION
**Pull request by AkshayPall**
A consistent button appears in the bottom-right once you scroll past the table of contents.

I'm very inexperienced with web development so feel free to help out haha. 

Questions:
- Should we use an icon instead of text? How would we represent the "table of contents" in a graphic?
- Should the button be centered to account for left-handed people (when viewing the site on mobile)?

Screen Shot 2021-06-13 at 10 11 19 PM
Screen Shot 2021-06-13 at 10 11 29 PM
Screen Shot 2021-06-13 at 10 11 40 PM

### CODE REVIEW

**Review by roxelo — APPROVED** [review]
LGTM, can merge once the merge conflict is resolved

**Comment by nsabahi** [comment]
For the text, I think having just "Table of Contents", "Back to TOC", or "Top ^" looks good (where "^" could be an up arrow icon).
- For a TOC graphic, perhaps something like this? (although I admit it kinda looks like a "menu" icon): 
image
- I'm also no front end expert haha :)

My only concern with having button in the middle would be if it obstructs with the content on the page.

Otherwise looks good! Thanks for implementing this.

**Comment by jaxonL** [comment]
> 
>     * For a TOC graphic, perhaps something like this? (although I admit it kinda looks like a "menu" icon):

piggy-backing on Nader's comment; i agree that the first thing that jumps up in my mind for a TOC icon would be something like that (or [some](https://fontawesome.com/v4.7/icon/list-alt) [alternatives](https://fontawesome.com/v4.7/icon/list) [here](https://fontawesome.com/v4.7/icon/list-ul))

(3 links, generally all looking like the image below: 
image )

about the button in the centre: i think it's a good idea for accessibility. it may block stuff, but we can scroll up so that it doesn't block content? i don't know if that makes sense.

another alternative would be to make it fade away after X amount of seconds, but make it reappear after scrolling up? (that would be another PR after this one is merged haha)

overall looks good! thank you for taking this up Akshay ><

**Comment by AkshayPall** [comment]
Okay so I added a menu icon but tbh after centering it, I feel maybe having a "^" icon would be better? what're your thoughts @jaxonL  @nsabahi 

Screen Shot 2021-06-14 at 5 40 26 PM
Screen Shot 2021-06-14 at 5 40 38 PM

**Comment by nsabahi** [comment]
Yea I agree. For the centered button an up-arrow or up-arrow with text like "Top" is better.

What do you think about having the text with the icon?

**Comment by AkshayPall** [comment]
> Yea I agree. For the centered button an up-arrow or up-arrow with text like "Top" is better.
> 
> What do you think about having the text with the icon?

I think having the ^ arrow with "top" would be redundant since they signify the same thing. But writing "Table of Contents" with the arrow would be too much? it's a difficult decision. 

I also wonder if "top" would even make sense because we're going not to the absolute top but rather the ToC and a reader might think "why would I want to go to the start" - will they remember that the table of contents is there? I guess so but maybe just writing "Table of Contents" or **"Sections"** would be more explicit? The latter would be more concise too

**Comment by nsabahi** [comment]
> I also wonder if "top" would even make sense

Yea I totally agree. Having ToC or Sections is definitely better. My only concern with _just_ an arrow was that it would go unnoticed or maybe unclear where the arrow goes to. But I'm no designer :P 

I think I'm leaning more towards having a small text with the icon (if we decide to include it), like what you suggested. But whatever we label the button should match with the title of that section. Like if we go with "Sections" we should probably change the title of that ToC to "Sections" as well.

**Comment by AkshayPall** [comment]
> Yea I totally agree. Having ToC or Sections is definitely better. My only concern with _just_ an arrow was that it would go unnoticed or maybe unclear where the arrow goes to. But I'm no designer :P
> 
> I think I'm leaning more towards having a small text with the icon (if we decide to include it), like what you suggested. But whatever we label the button should match with the title of that section. Like if we go with "Sections" we should probably change the title of that ToC to "Sections" as well.

I like this idea. I'll update this with "^ Sections" and update the ToC to say "sections".

**Comment by AkshayPall** [comment]
Screen Shot 2021-06-16 at 5 11 17 PM
Screen Shot 2021-06-16 at 5 11 28 PM
Screen Shot 2021-06-16 at 5 11 40 PM

**Comment by AkshayPall** [comment]
@roxelo @nsabahi @jaxonL  how's this?

**Comment by roxelo** [comment]
I like it!

File: README.md
Comment (jaxonL) [inline README.md]: no need to mention what it was previously imo? i think it's understandable if we just write `Sections` (documentation for ourselves)

**Review by jaxonL — APPROVED** [review]
tested on chrome 91.0, firefox 89.0, and edge (prolly outdated)

LGTM 👍

**Pull request merged**
